### ISSUE TITLE
How do I actually make the request using this lib

### ISSUE THREAD

**Issue by shuttzz** [lehh/nestjs-soap#15]
How to make a request for a method that is in wsdl through this lib, I did all the configuration, but I don't know how to actually make the request

**Comment by lehh** [comment]
Hello,

As stated in the docs, you need to make the request through the exposed soap client. For this, you can check out the [npm soap documentation](https://www.npmjs.com/package/soap) to better understand how to use the client.

If you need more help, please share more details with a sample code so I can help you better.

**Comment by RRGT19** [comment]
@shuttzz Here is a [topic](https://stackoverflow.com/questions/69752817/how-to-consume-soap-services-in-nestjs) on stackoverflow about it, maybe it can help you too.

**Comment by lehh** [comment]
Hello @RRGT19,

By viewing your answer in stackoverflow, I might need to let more clear that I'm using the [soap method async](https://www.npmjs.com/package/soap#clientmethodasyncargs-options---call-method-on-the-soap-service) in the example. I can also provide a repository with working examples later. 
That said, I don't see the point of adding examples of how to use the [soap](https://www.npmjs.com/package/soap) npm package if they already have a documentation for it. Nestjs-soap just exposes their client and, after that, you can just do whatever stated on their documentation. I can also try to make this clearer in the nestjs-soap docs.

TL;DR: nestjs-soap is only a **wrapper** for [soap](https://www.npmjs.com/package/soap) npm package. After exposing the client, you can just follow the [soap](https://www.npmjs.com/package/soap) docs.

Thanks for bringing this up. I will make these changes as soon as I can.
